.github/workflows/ockam-healthcheck.yml

```diff
@@ -0,0 +1,63 @@
+name: Ockam Healthcheck Container Release
+
+on:
+  pull_request:
+    paths:
+      - '.github/actions/ockam-healthcheck.yml'
+      - 'implementations/elixir/ockam/ockam_healthcheck/**'
+  push:
+    paths:
+      - '.github/actions/ockam-healthcheck.yml'
+      - 'implementations/elixir/ockam/ockam_healthcheck/**'
+    branches:
+      - develop
+
+permissions:
+  contents: read
```

Comment:
We'll need a `packages: write` permission so that we can push built image with GitHub action https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview